src/api/app/controllers/request_controller.rb

```diff
@@ -156,7 +156,7 @@ def request_create
 
     # cache the diff (in the backend)
     @req.bs_request_actions.each do |a|
-      a.delay.webui_infos
+      BsRequestActionWebuiInfosJob.perform_later(a.id)
```

Comment:
It was in the quick queue before. The name of the queue refers more to how quick the queue is worked off (the quick queue has 3 workers), instead of how quick the individual jobs are I guess.